ors-api/src/main/java/org/heigit/ors/api/ORSEnvironmentPostProcessor.java

```diff
@@ -1,30 +1,46 @@
 package org.heigit.ors.api;
 
+import org.apache.log4j.Logger;
+import org.heigit.ors.util.StringUtility;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.env.EnvironmentPostProcessor;
 import org.springframework.boot.env.YamlPropertySourceLoader;
 import org.springframework.core.env.ConfigurableEnvironment;
 import org.springframework.core.env.PropertySource;
+import org.springframework.core.env.StandardEnvironment;
 import org.springframework.core.io.FileSystemResource;
 
+import java.io.File;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
 public class ORSEnvironmentPostProcessor implements EnvironmentPostProcessor {
 
+    public static final String ORS_CONFIG_LOCATION_ENV = "ORS_CONFIG_LOCATION";
+    public static final String ORS_CONFIG_LOCATION_PROPERTY = "ors.config.location";
     private final YamlPropertySourceLoader loader = new YamlPropertySourceLoader();
 
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
-        // Override values from application.yml with contents of file in working directory. TODO: additional places?
-        String path = "ors-config.yml";
-        try {
-            List<PropertySource<?>> sources = this.loader.load("yml config", new FileSystemResource(path));
-            if (!sources.isEmpty()) {
-                environment.getPropertySources().addFirst(sources.get(0));
+        // Override values from application.yml with contents of custom config yml file.
+        // Later in array => higher precedence
+        List<String> configLocations = new ArrayList<>();
+        if (!StringUtility.isNullOrEmpty(System.getenv(ORS_CONFIG_LOCATION_ENV))) {
+            configLocations.add(System.getenv(ORS_CONFIG_LOCATION_ENV));
+        }
+        if (!StringUtility.isNullOrEmpty(System.getProperty(ORS_CONFIG_LOCATION_PROPERTY))) {
+            configLocations.add(System.getProperty(ORS_CONFIG_LOCATION_PROPERTY));
+        }
+        for (String path : configLocations) {
+            try {
+                List<PropertySource<?>> sources = this.loader.load("yml config", new FileSystemResource(path));
+                if (!sources.isEmpty()) {
+                    environment.getPropertySources().addAfter(StandardEnvironment.SYSTEM_ENVIRONMENT_PROPERTY_SOURCE_NAME, sources.get(0));
+                }
+            } catch (IllegalStateException | IOException ex) {
+                // Ignore yml file not present
```

Comment:
This logic could youse some testing imo or at least some debug (not info) level logging, to know that's being hit easily.